Assets/MixedRealityToolkit.SDK/Features/Utilities/Solvers/TapToPlace.cs

```diff
@@ -0,0 +1,344 @@
+        }
+
+        [SerializeField]
+        [Tooltip("If false, the game object to place will not change its rotation according to the surface hit.  The object will" +
+            " remain facing the camera while it is in the placing state.  If true, the object will rotate according to the surface normal" +
+            " if there is a hit.")]
+        private bool rotateAccordingToSurface = false;
+
+        /// <summary>
+        /// If false, the game object to place will not change its rotation according to the surface hit.  The object will 
+        /// remain facing the camera while it is in the placing state.  If true, the object will rotate according to the surface normal
+        /// if there is a hit."
+        /// </summary>
+        public bool RotateAccordingToSurface
+        {
+            get => rotateAccordingToSurface;
+            set => rotateAccordingToSurface = value;
+        }
+
+        [Tooltip("This event is triggered once at the start of the placing state.")]
+        /// <summary>
+        /// This event is triggered once at the start of the placing state.
+        /// </summary>
+        public UnityEvent OnPlacingStarted = new UnityEvent();
+
+        [Tooltip("This event is triggered once when the placing state has ended.")]
+        /// <summary>
+        /// This event is triggered once when the placing state has ended.
+        /// </summary>
+        public UnityEvent OnPlacingStopped = new UnityEvent();
+
+        /// <summary>
+        /// Get the game object layer before the object is placed.  When an object is in the placing state, the layer
+        /// is changed to IgnoreRaycast and then back to the original layer.
+        /// </summary>
+        public int GameObjectLayer { get; protected set; }
+
+        protected internal bool IsColliderPresent => gameObject != null ? gameObject.GetComponent<Collider>() != null : false;
+
+        private const int IgnoreRaycastLayer = 2;
+
+        // The current ray is based on the TrackedTargetType (Controller Ray, Head, Hand Joint)
+        protected RayStep CurrentRay;
+
+        protected bool DidHitSurface;
+
+        protected RaycastHit CurrentHit;
+
+        protected float TimeSinceClick = 0;
+
+        #region MonoBehaviour Implementation
+        protected override void Start()
+        {
+            // Solver base class
+            base.Start();
+
+            if (!IsColliderPresent)
+            {
+                Debug.LogError("The GameObjectToPlace does not have a collider attached, please attach a collider");
+            }
+
+            SurfaceNormalOffset = gameObject.GetComponent<Collider>().bounds.extents.z;
+
+            // Store the initial game object layer
+            GameObjectLayer = gameObject.layer;
+
+            if (AutoStart)
+            {
+                StartPlacement();
+                OnPlacingStarted?.Invoke();
+            }
+            else
+            {
+                SolverHandler.UpdateSolvers = false;
```

Comment:
I think this was discussed on a previous thread which got resolved, but I still think that you don't want to kill off all solvers - you just don't want to do anything in this particular one.

Reply:
Reading our code a bit more, I see there's already a precedent for calling SolverHandler.UpdateSolvers = false in individual solvers. So this component is technically following an existing pattern. I'd recommend staying consistent with other solvers and maybe filing a bug to fix this.